{"@t":"2023-03-03T07:05:49.0679220Z","@mt":"Registered 3 endpoints in 182 milliseconds.","SourceContext":"FastEndpoints.StartupTimer","Application":"messages-service"}
{"@t":"2023-03-03T07:05:49.2057551Z","@mt":"No validators found in the system!","SourceContext":"FastEndpoints.Swagger.ValidationSchemaProcessor","Application":"messages-service"}
{"@t":"2023-03-03T07:05:49.2896634Z","@mt":"Connecting to Redis endpoints: {Endpoints}. Using Server Name: {ServerName}","Endpoints":"localhost","ServerName":"DESKTOP-G3RIR5H_242954e80a904ee8bcf1787dfe6323ae","EventId":{"Id":1,"Name":"ConnectingToEndpoints"},"SourceContext":"Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager","Application":"messages-service"}
{"@t":"2023-03-03T07:05:49.4738821Z","@mt":"Connected to Redis.","EventId":{"Id":2,"Name":"Connected"},"SourceContext":"Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager","Application":"messages-service"}
{"@t":"2023-03-03T07:05:51.2731042Z","@mt":"Configured endpoint {Endpoint}, Consumer: {ConsumerType}","Endpoint":"AppUserLoggedInEvent-Messages","ConsumerType":"Messages.API.Consumers.AppUserLoggedInConsumer","SourceContext":"MassTransit","Application":"messages-service"}
{"@t":"2023-03-03T07:05:51.5235229Z","@mt":"Now listening on: {address}","address":"https://localhost:6002","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T07:05:51.5243681Z","@mt":"Now listening on: {address}","address":"http://localhost:5002","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T07:05:51.5250549Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T07:05:51.5255777Z","@mt":"Hosting environment: {EnvName}","EnvName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T07:05:51.5262409Z","@mt":"Content root path: {ContentRoot}","ContentRoot":"C:\\Users\\jenki\\source\\app\\solar-engineer\\src\\Services\\Messages.API","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T07:05:51.6645181Z","@mt":"Bus started: {HostAddress}","HostAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service"}
{"@t":"2023-03-03T08:07:01.6505666Z","@mt":"Connection to Redis failed.","@l":"Error","@x":"StackExchange.Redis.RedisConnectionException: SocketClosed (ReadEndOfStream, last-recv: 0) on localhost:6379/Subscription, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 0, last-read: 0s ago, last-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.2.4.27433","EventId":{"Id":9,"Name":"ConnectionFailed"},"SourceContext":"Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager","Application":"messages-service","ExceptionDetail":{"Data":{"Redis-FailureType":"SocketClosed","Redis-EndPoint":"localhost:6379","Redis-Origin":"ReadFromPipe","Redis-Outstanding-Responses":"0","Redis-Last-Read":"0s ago","Redis-Last-Write":"0s ago","Redis-Keep-Alive":"60s","Redis-Previous-Physical-State":"ConnectedEstablished","Redis-Manager":"8 of 10 available","Redis-Inbound-Bytes":"0","Redis-Inbound-Pipe-Bytes":"0","Redis-Outbound-Pipe-Bytes":"0","Redis-Last-Heartbeat":"0s ago","Redis-Last-Multiplexer-Heartbeat":"0s ago","Redis-Last-Global-Heartbeat":"0s ago","Redis-Version":"2.2.4.27433"},"HResult":-2146233088,"Message":"SocketClosed (ReadEndOfStream, last-recv: 0) on localhost:6379/Subscription, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 0, last-read: 0s ago, last-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.2.4.27433","Source":null,"FailureType":"SocketClosed","CommandStatus":"Unknown","Type":"StackExchange.Redis.RedisConnectionException"}}
{"@t":"2023-03-03T08:07:09.2638894Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.IO.IOException: connection.start was never received, likely due to a network timeout\r\n   at RabbitMQ.Client.Framing.Impl.Connection.StartAndTune()\r\n   at RabbitMQ.Client.Framing.Impl.Connection.Open(Boolean insist)\r\n   at RabbitMQ.Client.Framing.Impl.Connection..ctor(IConnectionFactory factory, Boolean insist, IFrameHandler frameHandler, String clientProvidedName)\r\n   at RabbitMQ.Client.Framing.Impl.Connection..ctor(IConnectionFactory factory, Boolean insist, IFrameHandler frameHandler, ArrayPool`1 memoryPool, String clientProvidedName)\r\n   at RabbitMQ.Client.Framing.Impl.ProtocolBase.CreateConnection(IConnectionFactory factory, Boolean insist, IFrameHandler frameHandler, ArrayPool`1 memoryPool, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.IO.IOException","HResult":-2146232800,"Message":"connection.start was never received, likely due to a network timeout","Source":"RabbitMQ.Client","TargetSite":"Void StartAndTune()"},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:07:20.0544407Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:07:35.0255297Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:07:54.9014129Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:08:23.3369200Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:08:58.4338934Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:09:33.5114292Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:10:08.6165174Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:10:43.7218430Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:11:18.7768950Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:11:53.8510452Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:12:28.9268385Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:13:04.0128589Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:13:39.0716844Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:14:14.1637579Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:14:49.2253811Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:14:49.8452240Z","@mt":"Application is shutting down...","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:14:49.8841659Z","@mt":"Bus stopped: {HostAddress}","HostAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service"}
{"@t":"2023-03-03T08:15:00.5022753Z","@mt":"Registered 3 endpoints in 124 milliseconds.","SourceContext":"FastEndpoints.StartupTimer","Application":"messages-service"}
{"@t":"2023-03-03T08:15:00.5765219Z","@mt":"No validators found in the system!","SourceContext":"FastEndpoints.Swagger.ValidationSchemaProcessor","Application":"messages-service"}
{"@t":"2023-03-03T08:15:00.5983632Z","@mt":"Connecting to Redis endpoints: {Endpoints}. Using Server Name: {ServerName}","Endpoints":"localhost","ServerName":"DESKTOP-G3RIR5H_a1a8ea5db6c94e4299c8f6b7813422c4","EventId":{"Id":1,"Name":"ConnectingToEndpoints"},"SourceContext":"Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager","Application":"messages-service"}
{"@t":"2023-03-03T08:15:02.2620374Z","@mt":"Configured endpoint {Endpoint}, Consumer: {ConsumerType}","Endpoint":"AppUserLoggedInEvent-Messages","ConsumerType":"Messages.API.Consumers.AppUserLoggedInConsumer","SourceContext":"MassTransit","Application":"messages-service"}
{"@t":"2023-03-03T08:15:02.6196927Z","@mt":"Now listening on: {address}","address":"https://localhost:6002","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:15:02.6210454Z","@mt":"Now listening on: {address}","address":"http://localhost:5002","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:15:02.6225901Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:15:02.6236909Z","@mt":"Hosting environment: {EnvName}","EnvName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:15:02.6249651Z","@mt":"Content root path: {ContentRoot}","ContentRoot":"C:\\Users\\jenki\\source\\app\\solar-engineer\\src\\Services\\Messages.API","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:15:04.7438463Z","@mt":"Error connecting to Redis.","@l":"Error","@x":"StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s). UnableToConnect on localhost:6379/Interactive, Initializing/NotStarted, last: NONE, origin: BeginConnectAsync, outstanding: 0, last-read: 2s ago, last-write: 2s ago, keep-alive: 60s, state: Connecting, mgr: 10 of 10 available, last-heartbeat: never, global: 4s ago, v: 2.2.4.27433\r\n   at StackExchange.Redis.ConnectionMultiplexer.ConnectImplAsync(ConfigurationOptions configuration, TextWriter log) in /_/src/StackExchange.Redis/ConnectionMultiplexer.cs:line 861\r\n   at Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisOptions.ConnectAsync(TextWriter log)\r\n   at Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager`1.EnsureRedisServerConnection()","EventId":{"Id":14,"Name":"ErrorConnecting"},"SourceContext":"Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager","Application":"messages-service","ExceptionDetail":{"HResult":-2146233088,"Message":"It was not possible to connect to the redis server(s). UnableToConnect on localhost:6379/Interactive, Initializing/NotStarted, last: NONE, origin: BeginConnectAsync, outstanding: 0, last-read: 2s ago, last-write: 2s ago, keep-alive: 60s, state: Connecting, mgr: 10 of 10 available, last-heartbeat: never, global: 4s ago, v: 2.2.4.27433","Source":"StackExchange.Redis","TargetSite":"Void MoveNext()","FailureType":"UnableToConnect","CommandStatus":"Unknown","Type":"StackExchange.Redis.RedisConnectionException"}}
{"@t":"2023-03-03T08:15:06.5728356Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:15:17.3341023Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:15:30.4072817Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:15:50.0587643Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:16:23.9123211Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:16:59.0085898Z","@mt":"Connection Failed: {InputAddress}","@l":"Warning","@x":"RabbitMQ.Client.Exceptions.BrokerUnreachableException: None of the specified endpoints were reachable\r\n ---> System.AggregateException: One or more errors occurred. (Connection failed)\r\n ---> RabbitMQ.Client.Exceptions.ConnectFailureException: Connection failed\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\r\n   at System.Threading.Tasks.ValueTask.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)\r\n--- End of stack trace from previous location ---\r\n   at RabbitMQ.Client.Impl.TcpClientAdapter.ConnectAsync(String host, Int32 port)\r\n   at RabbitMQ.Client.Impl.TaskExtensions.TimeoutAfter(Task task, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectOrFail(ITcpClient socket, AmqpTcpEndpoint endpoint, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingAddressFamily(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout, AddressFamily family)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler.ConnectUsingIPv4(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan timeout)\r\n   at RabbitMQ.Client.Impl.SocketFrameHandler..ctor(AmqpTcpEndpoint endpoint, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.Framing.Impl.IProtocolExtensions.CreateFrameHandler(IProtocol protocol, AmqpTcpEndpoint endpoint, ArrayPool`1 pool, Func`2 socketFactory, TimeSpan connectionTimeout, TimeSpan readTimeout, TimeSpan writeTimeout)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateFrameHandler(AmqpTcpEndpoint endpoint)\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.EndpointResolverExtensions.SelectOne[T](IEndpointResolver resolver, Func`2 selector)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   --- End of inner exception stack trace ---\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IEndpointResolver endpointResolver, String clientProvidedName)\r\n   at RabbitMQ.Client.ConnectionFactory.CreateConnection(IList`1 hostnames, String clientProvidedName)\r\n   at MassTransit.RabbitMqTransport.ConnectionContextFactory.CreateConnection(ISupervisor supervisor) in /_/src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/ConnectionContextFactory.cs:line 86","InputAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service","ExceptionDetail":{"HResult":-2146232800,"Message":"None of the specified endpoints were reachable","Source":"RabbitMQ.Client","TargetSite":"RabbitMQ.Client.IConnection CreateConnection(RabbitMQ.Client.IEndpointResolver, System.String)","InnerException":{"Type":"System.AggregateException","HResult":-2146233088,"Message":"One or more errors occurred. (Connection failed)","Source":"RabbitMQ.Client","TargetSite":"T SelectOne[T](RabbitMQ.Client.IEndpointResolver, System.Func`2[RabbitMQ.Client.AmqpTcpEndpoint,T])","InnerException":{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"},"InnerExceptions":[{"HResult":-2146233088,"Message":"Connection failed","Source":"RabbitMQ.Client","TargetSite":"Void ConnectOrFail(RabbitMQ.Client.ITcpClient, RabbitMQ.Client.AmqpTcpEndpoint, System.TimeSpan)","InnerException":{"Type":"System.Net.Sockets.SocketException","HResult":-2147467259,"Message":"No connection could be made because the target machine actively refused it.","Source":"System.Net.Sockets","TargetSite":"Void ThrowException(System.Net.Sockets.SocketError, System.Threading.CancellationToken)","SocketErrorCode":"ConnectionRefused","SocketErrorCodeMessage":"The remote host is actively refusing a connection."},"Type":"RabbitMQ.Client.Exceptions.ConnectFailureException"}]},"Type":"RabbitMQ.Client.Exceptions.BrokerUnreachableException"}}
{"@t":"2023-03-03T08:17:09.3687645Z","@mt":"Application is shutting down...","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:17:09.3828547Z","@mt":"Failed to stop bus: {Address} ({Reason})","@l":"Warning","Address":"rabbitmq://localhost/DESKTOPG3RIR5H_MessagesAPI_bus_dy3oyyn7aeybmiumbdptzx5dn1?temporary=true","Reason":"Not Started","SourceContext":"MassTransit","Application":"messages-service"}
{"@t":"2023-03-03T08:17:17.0338233Z","@mt":"Registered 3 endpoints in 129 milliseconds.","SourceContext":"FastEndpoints.StartupTimer","Application":"messages-service"}
{"@t":"2023-03-03T08:17:17.1092391Z","@mt":"No validators found in the system!","SourceContext":"FastEndpoints.Swagger.ValidationSchemaProcessor","Application":"messages-service"}
{"@t":"2023-03-03T08:17:17.1311515Z","@mt":"Connecting to Redis endpoints: {Endpoints}. Using Server Name: {ServerName}","Endpoints":"localhost","ServerName":"DESKTOP-G3RIR5H_d8219215ed094c0199bd519da159c918","EventId":{"Id":1,"Name":"ConnectingToEndpoints"},"SourceContext":"Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager","Application":"messages-service"}
{"@t":"2023-03-03T08:17:17.2361699Z","@mt":"Connected to Redis.","EventId":{"Id":2,"Name":"Connected"},"SourceContext":"Microsoft.AspNetCore.SignalR.StackExchangeRedis.RedisHubLifetimeManager","Application":"messages-service"}
{"@t":"2023-03-03T08:17:18.5239553Z","@mt":"Configured endpoint {Endpoint}, Consumer: {ConsumerType}","Endpoint":"AppUserLoggedInEvent-Messages","ConsumerType":"Messages.API.Consumers.AppUserLoggedInConsumer","SourceContext":"MassTransit","Application":"messages-service"}
{"@t":"2023-03-03T08:17:18.8054075Z","@mt":"Now listening on: {address}","address":"https://localhost:6002","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:17:18.8062374Z","@mt":"Now listening on: {address}","address":"http://localhost:5002","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:17:18.8068728Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:17:18.8073415Z","@mt":"Hosting environment: {EnvName}","EnvName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:17:18.8078609Z","@mt":"Content root path: {ContentRoot}","ContentRoot":"C:\\Users\\jenki\\source\\app\\solar-engineer\\src\\Services\\Messages.API","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T08:17:18.9405800Z","@mt":"Bus started: {HostAddress}","HostAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service"}
{"@t":"2023-03-03T11:10:45.6628517Z","@mt":"Application is shutting down...","SourceContext":"Microsoft.Hosting.Lifetime","Application":"messages-service"}
{"@t":"2023-03-03T11:10:45.7596016Z","@mt":"Bus stopped: {HostAddress}","HostAddress":"rabbitmq://localhost/","SourceContext":"MassTransit","Application":"messages-service"}
